src/assets/i18n/de_DE.json

```diff
@@ -297,6 +298,20 @@
   "account.organization.cost_center_management.create.confirmation": "Die Kostenstelle \"{{0}}\" wurde erstellt.",
   "account.organization.cost_center_management.delete.confirmation": "Die Kostenstelle wurde gelöscht.",
   "account.organization.cost_center_management.update.confirmation": "Die Kostenstelle \"{{0}}\" wurde geändert.",
+  "account.organization.hierarchies": "Struktur des Unternehmens",
+  "account.organization.hierarchies.grouppath.tooltip.close": "Ausblenden",
+  "account.organization.hierarchies.grouppath.tooltip.open": "Alle Gruppen anzeigen",
+  "account.organization.hierarchies.groups.button.add.label": "Gruppe hinzufügen",
+  "account.organization.hierarchies.groups.delete.confirmation": "Die Gruppe mit der ID \"{{0}}\" wurde gelöscht.",
+  "account.organization.hierarchies.groups.new.button.label": "Anlegen",
+  "account.organization.hierarchies.groups.new.confirmation": "Die Gruppe \"{{0}}\" wurde erstellt.",
+  "account.organization.hierarchies.groups.new.message": "Erstellen Sie eine neue Gruppe, um Ihre Unternehmensstruktur abzubilden. Sie können ihr Benutzer zuordnen.",
+  "account.organization.hierarchies.groups.select.placeholder": "Gruppe auswählen",
+  "account.organization.hierarchies.link": "Struktur des Unternehmens",
+  "account.organization.hierarchies.remove_dialog.label_no": "Abbrechen",
+  "account.organization.hierarchies.remove_dialog.label_yes": "Löschen",
+  "account.organization.hierarchies.remove_dialog.message_line2": "Möchten Sie wirklich die Gruppe \"{{0}}\" löschen?",
```

Comment:
"Möchten Sie die Gruppe \"{{0}}\" wirklich löschen?",

Maybe also use German quotation marks instead?

Reply:
Won't start using German quotation marks here now. Seems to be fine as is.